monkey/agent_plugins/payloads/cryptominer/src/cryptominer_options.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import Field, conint
+
+from common.base_models import InfectionMonkeyBaseModel
+
+
+class CryptominerOptions(InfectionMonkeyBaseModel):
+    simulation_time: float = Field(
+        title="Simulation time",
+        description="The time (in seconds) for which the cryptomining simulation should run"
+        " on each machine",
+        default=0,
```

Comment:
If the user enables the cryptominer payload, I think he will want to run it for some time as default. Although I am on two minds

Reply:
"Indefinitely" should be an option. The easiest way to accomplish that is to say that `0` means "run until I press the stop button."

5 minutes seems like a reasonable default.

Reply:
On second thought, maybe that's a dangerous idea. Maybe it shoudn't run indefinitely. Since there's no maximum, the user could set this up very high if they want it to run indefinitely, but that's unambiguously their fault.

5 minutes still seems like a reasonable default.